main/wpa_supplicant/wpa_supplicant.initd

```diff
@@ -2,25 +2,25 @@
 # Copyright (c) 2009 Roy Marples <roy@marples.name>
 # All rights reserved. Released under the 2-clause BSD license.
 
+supervisor=supervise-daemon
+
+name="WPA Supplicant"
+description="Wi-Fi Protected Access client and IEEE 802.1X supplicant"
+
 command=/sbin/wpa_supplicant
 default_conf=/etc/wpa_supplicant/wpa_supplicant.conf
 wpa_supplicant_if=${wpa_supplicant_if:+-i}$wpa_supplicant_if
-command_args="$wpa_supplicant_args -B $wpa_supplicant_if"
-name="WPA Supplicant Daemon"
-
-default_conf=/etc/wpa_supplicant/wpa_supplicant.conf
+command_args="$wpa_supplicant_args -c$wpa_supplicant_conf $wpa_supplicant_if"
```

Comment:
Reintroducing `-c$wpa_supplicant_conf` and removing the `default_conf` will break the things i fixed with 978f22f7b0a0c5fc92eb9f49ecc7c08dd962bf3b. We shoudl only add `-c$wpa_supplicant_conf` if its either explicitly set  in conf.d or the default `/etc/wpa_supplicant/wpa_supplicant.conf` exists. We do this in `start_pre()`.

Reply:
Fixed, the default_conf is 3 lines above